src/main/cli/settings.c

```diff
@@ -642,7 +643,10 @@ const clivalue_t valueTable[] = {
 
 // PG_COMPASS_CONFIG
 #ifdef USE_MAG
-    { "align_mag",                  VAR_UINT8  | HARDWARE_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_ALIGNMENT }, PG_COMPASS_CONFIG, offsetof(compassConfig_t, mag_align) },
+    { "align_mag",                  VAR_UINT8  | HARDWARE_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_ALIGNMENT }, PG_COMPASS_CONFIG, offsetof(compassConfig_t, mag_alignment) },
+    { "mag_align_roll",             VAR_INT16  | HARDWARE_VALUE, .config.minmax = { -360, 360 }, PG_COMPASS_CONFIG, offsetof(compassConfig_t, mag_customAlignment.values.roll) },
```

Comment:
there were some discussion on this, I also think it may be useful to have better than 1 degree resolution, there are some use cases:
1) where boards with multiple gyros have gyros that are not well placed/assembled.
2) boards with external gyros, e.g. the ones in potted cases.
3) board with with gyros connected via flex cables.

Reply:
done code now uses DECIDGREES, a change to CENTIDEGREES (signed) would overflow int16_t.  i.e. +/-36000 is > +/- 32756.